### PR TITLE
Update CODEOWNERS

### PR DESCRIPTION
**Pull request by millsp**

### CODE REVIEW

**Review by jkomyno — APPROVED** [review]

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -2,4 +2,4 @@
 
 # These owners will be the default owners for everything in
 # the repo. Unless a later match takes precedence,
-*       @Jolg42 @millsp @aqrln
+*       @Jolg42 @SevInf @jkomyno @druue
```
Comment (aqrln) [inline .github/CODEOWNERS:5]: It seems that they don't have write access to the repo

Comment (aqrln) [reply .github/CODEOWNERS:5]: image

image

**Review by Jolg42 — APPROVED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (Jolg42) [reply .github/CODEOWNERS:5]: I asked @apolanc https://prisma-company.slack.com/archives/C058VM009HT/p1709821212095269

Comment (SevInf) [reply .github/CODEOWNERS:5]: > Make sure @SevInf exists

Thanks for existential crisis, Github!

**Pull request merged**